sdk/spring/azure-spring-boot-samples/azure-spring-cloud-appconfiguration-conversion-sample-initial/README.md

```diff
@@ -86,9 +86,9 @@ In this section, you clone a containerized Spring Boot application and test it l
 1. Replace below properties in `application.properties` with information from your database.
 
    ```properties
-   azure.cosmosdb.uri=your-cosmosdb-uri
-   azure.cosmosdb.key=your-cosmosdb-key
-   azure.cosmosdb.database=your-cosmosdb-databasename
+   azure.cosmos.uri=your-cosmosdb-uri
+   azure.cosmos.key=your-cosmosdb-key
+   azure.cosmos.database=your-cosmosdb-databasename
 
```

Comment:
you update `your-cosmosdb-uri` to `your-cosmos-uri` in application.properties, I think it should be updated here too. key/database is same

Reply:
Got it.